content/cloud-docs/api-docs/run-tasks-integration.mdx

```diff
@@ -0,0 +1,147 @@
+---
+page_title: Run Tasks Integration - API Docs - Terraform Cloud and Terraform Enterprise
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# Run Tasks Integration API
+
+-> Note: As of September 2021, Run Tasks are available only as a beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization.
+
+[Run tasks](/cloud-docs/workspaces/settings/run-tasks) allow Terraform Cloud to interact with external systems at specific points in the Terraform Cloud run lifecycle.
+Run tasks are reusable configurations that you can attach to any workspace in an organization.
+This page lists the API endpoints which are used when triggering a Run Task and the expected response from the integration.
+
+You can find more information about managing Run Tasks within Terraform Cloud in [Run Tasks Management](/cloud-docs/api-docs/run-tasks) documentation.
+
+## Run Task Request
+
+When a run reaches the appropriate phase and a run task is triggered, Terraform Cloud will send a request to the Run Task's URL.
```

Comment:
Okay one thing I'm noticing is that we're capitalizing Run Task here. But it's not capitalized throughout the documentation, and I feel like it's the general convention that we don't capitalize features. For example, we don't capitalized "variable sets" or the "private registry". https://www.terraform.io/cloud-docs/integrations/run-tasks

I think we should un-capitalize it everywhere except the title. Thoughts?

Reply:
No, that's fair.  I think I capitilised it because that's it's name.  I'll go through the changes and lowercase it.

Reply:
Added commit https://github.com/hashicorp/terraform-website/pull/2193/commits/5b406d04930533555f2007a0b7e9b9096818f334